src/multidim/box_macro.jl

```diff
@@ -0,0 +1,42 @@
+using ValidatedNumerics
+
+
+"""Macro to make a version of a multidimensional function that acts on
+an `IntervalBox` and returns an `IntervalBox`.
+Both the original n-argument function and the `IntervalBox` version are defined.
+
+Example:
+
+    @box f(x, y) = (x + y, x - y)
+
+    X = IntervalBox(1..1, 2..2)
+    f(X)
+
+(No significant error checking is performed!)
+"""
+
+macro box(ex)
```

Comment:
I wanted to use the name `Box`, instead of `IntervalBox`, but until recently `Box` was an exported internal name. However, maybe `IntervalBox` is a better name. I will change `@box` to `@intervalbox`